### DISCUSSION TITLE
Translate block content

### DISCUSSION THREAD

**Discussion by rogmauri** [woocommerce/woocommerce#43542] [Checkout Flow]
How to translate the content of blocks, such as titles, subtitles and details into Portuguese (Brazil).
![Capturar](https://github.com/woocommerce/woocommerce/assets/42703799/7b2e13ca-23e4-4593-89c7-8166c24f25be)

**Comment by rrennick** [comment]
See https://github.com/woocommerce/woocommerce/blob/trunk/docs/localization-translation/translating-woocommerce.md

**Reply by sennevdp** [reply]
The document you refer to is about the default Woocommerce plugin translation, but we are unable to translate the new Checkout and Cart blocks (all other WooCommerce string translation work just fine). When using a translation plugin such as Loco Translate, all strings appear to be correctly translated (to Dutch in my case). However, on the Cart and Checkout pages, most strings are still shown in English. So the same problem as @rogmauri mentions above in this treath.

**Comment by rrennick** [comment]
> The document you refer to is about the default Woocommerce plugin translation, but we are unable to translate the new Checkout and Cart blocks (all other WooCommerce string translation work just fine).

The strings for both PHP and blocks are included in that translation file.

> When using a translation plugin such as Loco Translate, all strings appear to be correctly translated (to Dutch in my case).

This is an issue you will need to take up with Loco Translate support. `js` translations are stored in the `wp-content/languages/plugins` folder and are not filterable via the WP translation filters available in PHP.

**Reply by sennevdp** [reply]
Thanks for your input, that clears things up!

**Reply by nolan450** [reply]
Hey, @sennevdp Did you resolve your issue?
